web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -1286,8 +1286,8 @@ class AppVersionHistory extends Component {
             </select>
           </div>
         </div>
-        {allVersions?.map((version) =>
-          this.renderAppVersionHistoryRow(version)
+        {allVersions?.map((version, index) =>
+          this.renderAppVersionHistoryRow(version, index)
```

Comment:
This is a minor refactor that gets rid of a warning from a rendered list with duplicate keys.